test/repro/reg_test_5985_admin_caching_sha2_full_auth.bash

```
@@ -0,0 +1,242 @@
+if [ "$(sha2cached)" != "0" ]; then
+	die "Client_Connections_sha2cached is '$(sha2cached)', not 0: the cleartext cache is
+       already warm, so a successful login here would prove nothing. Re-run with
+       COLD_START=1 to recreate the instance."
+fi
+
+restore() {
+	adm "UPDATE global_variables SET variable_value='${BASE_CREDS}' WHERE variable_name='admin-admin_credentials';
+	     LOAD ADMIN VARIABLES TO RUNTIME;
+	     UPDATE global_variables SET variable_value='mysql_native_password' WHERE variable_name='mysql-default_authentication_plugin';
+	     LOAD MYSQL VARIABLES TO RUNTIME;" >/dev/null
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)